Source/Godunov.cpp

```diff
@@ -510,7 +563,8 @@ pc_umeth_2D(
   amrex::ParallelFor(
     yflxbx, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
       pc_cmpflx(
-        i, j, k, bcly, bchy, dly, dhy, qymarr, qyparr, fyarr, q2, qaux, cdir);
+        i, j, k, bcly, bchy, dly, dhy, qymarr, qyparr, fyarr, gdtemp, qaux,
```

Comment:
did you mena to switch from q2 to gdtemp?

Reply:
Yeah, although I will have to double check because I'm seeing unexpected errors. But in 3D all the equivalent calls put this into a temporary fab because it shouldn't be needed

Reply:
Turns out that was the issue, good catch